CONTRIBUTING.md

```diff
@@ -3,12 +3,16 @@ Pull requests which drastically alter the structure or architecture of the code
 This is to maintain "sanity" whilst going back through code, comparing to the original MIPS disassembled output.
 Best pull requests are which resolve bugs, finish existing decompiled methods, improve readability, introduce new decompiled methods and improve code documentation.
 
+Also it's good to see which issues have more priority to work with here:
+https://github.com/SoapyMan/REDRIVER2/contribute
+
 # Compiling
-Project **must** be built only for 32 bit platform because there are 32 bit pointers everywhere
-Use [premake5](https://github.com/premake/premake-core/releases/download/v5.0.0-alpha15/premake-5.0.0-alpha15-windows.zip) to generate project files.
+Project **must** be built only for **32 bit** platform because there are 32 bit pointers everywhere (and use of *long* which with GCC can be a 64 bit numbers)
+
+First use [premake5](https://github.com/premake/premake-core/releases/download/v5.0.0-alpha15/premake-5.0.0-alpha15-windows.zip) to generate project files.
 
-Dependencies (Windows):
-- [SDL2](https://www.libsdl.org/release/SDL2-devel-2.0.12-VC.zip)
+Dependencies (links for Windows):
+- [SDL 2](https://www.libsdl.org/release/SDL2-devel-2.0.12-VC.zip)
 - [GLEW](https://netix.dl.sourceforge.net/project/glew/glew/2.1.0/glew-2.1.0-win32.zip)
 - [OpenAL-Soft](https://kcat.strangesoft.net/openal-binaries/openal-soft-1.20.1-bin.zip)
```

Comment:
Don't forget to update this link: https://openal-soft.org/openal-binaries/openal-soft-1.20.1-bin.zip